src/api/nodeRpcCall.js

```diff
@@ -1305,6 +1305,25 @@ const getBlockEvents = async (blockHash) => {
   }
 }
 
+const getMotions = async () => {
+  const api = await getApi();
+  const proposals = await api.query.council.proposals()
+  
+  return await Promise.all(
+    proposals.map(async proposal => {
+      const [proposalOf, voting] = await api.queryMulti([
```

Comment:
So we're fetching N proposals and then we do another call per proposal, effectively doing N+1 calls. This is known as "N+1 query problem".

While this is OK here , as I don't expect many proposals, in the future in other cases consider one of these 2 solutions:
1. Fetch all proposals + all proposalOf (using .entries()) + all voting (using .entries()) and join them client side, which would make only 1 call to node, or
2. Fetch all proposals and then fetch all proposalOf and voting for these proposals using `.multi()`. As .multi returns result sorted the same way as incoming list, it's trivial to join them. That would only do 3 calls to node.